modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/config/StorageTarget.scala

```diff
@@ -194,6 +194,8 @@ object StorageTarget {
       val AwsRegionsWithSegment =
         List("us-east-2", "us-east-1-gov", "ca-central-1", "eu-west-2", "ap-northeast-1", "ap-south-1")
       val GcpRegions = List("us-central1", "europe-west2", "europe-west4")
+
+      // TODO do we have to change here anything?
```

Comment:
I think no, because this is about the target warehouse.  It's not about where the pipeline is running.

@peel is the expert on these host names though.